src/resource.js

```diff
@@ -124,6 +152,16 @@ class Resource extends BaseResource {
     }, {})
     return new ValidationError(`${this.name()} validation failed`, errors)
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  convertMongooseObject(mongooseObj) {
+    const obj = mongooseObj.toObject()
+
+    // By default the _id field is an ObjectID, one of MongoDB's BSON
+    // We have to convert this field to string for the flatten record params
```

Comment:
I would change this to pasive:

`Id field has to be converted to string, because.... (I dont understand: "for the flatten record params")